src/EPR.Calculator.Frontend/Constants/StaticHelpers.cs

```diff
@@ -8,5 +8,6 @@ public class StaticHelpers
         public const string Path = @"wwwroot/assets/SchemeParameterTemplates/SchemeParameterTemplate.v0.1.xlsx";
         public const string FileName = "test.csv";
         public const string MimeType = "test/csv";
+        public const string Index = "Index";
```

Comment:
Sorry  I should have been more clear

I was expecting a Constant called ViewNames - With all the view names may be an public Static Class constant.